### PR TITLE
ROC-5409 Create smaller PRs

### PR DESCRIPTION
**Pull request by robertparkinson**
### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/app/forms/validation/validationErrors.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export class ValidationErrors {
   static readonly AMOUNT_INVALID_DECIMALS: string = 'Enter a valid amount, maximum two decimal places'
   static readonly NON_NEGATIVE_NUMBER_REQUIRED: string = 'Don’t enter a negative number'
   static readonly POSITIVE_NUMBER_REQUIRED: string = 'Enter a number higher than 0'
+  static readonly BELOW_OR_EQUAL_TO_100_REQUIRED: string = 'Enter a number lesser than or equal to 100'
```
Comment (andrewwa-kainos) [inline src/main/app/forms/validation/validationErrors.ts]: "less than"

File: src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk [hunk 2]
```diff
@@ -144,6 +144,23 @@
...
+    <p>{{ t('You’ve both agreed to try mediation.') }}</p>
+  {% endif %}
+{% endmacro %}
+
+{% macro claimRejectedDefendantDefenceFreeMediationYESForClaimantDetails(claim) %}
+  <div class="status-block">
+    <h2 class="heading-small">{{ t('You’ve rejected the defendant’s response.') }}</h2>
+    <p>{{ t('You’ve both agreed to try mediation. The Small Claims Mediation Service will contact you to arrange an appointment.') }}</p>
+    {{ externalLink('Find out how mediation works', 'https://www.gov.uk/government/publications/small-claims-mediation-service-ex730') }}
+    <p>{{ _downloadResponseLink(claim, 'Download your response') }}</p>
```
Comment (andrewwa-kainos) [inline src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk]: Isn't this the defendant's response link? (If so, check the rest of this file!)

File: src/main/features/directions-questionnaire/forms/models/reportRow.ts [hunk 3]
```diff
@@ -17,12 +21,14 @@ export class ReportRow extends MultiRowFormItem {
...
   @IsDefined({ message: ValidationErrors.NAME_REQUIRED })
   @IsNotBlank({ message: ValidationErrors.NAME_REQUIRED })
+  @MaxLength(ValidationConstraints.STANDARD_TEXT_INPUT_MAX_LENGTH, { message: DefaultValidationErrors.TEXT_TOO_LONG })
   expertName?: string
 
   @ValidateIf(o => !!o.expertName)
   @IsDefined({ message: ValidationErrors.DATE_REQUIRED })
   @IsValidLocalDate({ message: ValidationErrors.VALID_DATE_REQUIRED })
   @IsPastDate({ message: ValidationErrors.PAST_DATE_REQUIRED })
+
```
Comment (andrewwa-kainos) [inline src/main/features/directions-questionnaire/forms/models/reportRow.ts]: I would prefer to keep all the annotations together

File: src/main/features/directions-questionnaire/routes/permission-for-expert.ts [hunk 4]
```diff
@@ -28,17 +33,22 @@ export default express.Router()
...
+        if (form.model.option.option === YesNoOption.NO.option && draft.document.permissionForExpert && draft.document.permissionForExpert.option && draft.document.permissionForExpert.option.option === YesNoOption.YES.option) {
+          draft.document.expertEvidence = new ExpertEvidence()
+          draft.document.whyExpertIsNeeded = new WhyExpertIsNeeded()
+        }
         draft.document.permissionForExpert = form.model
 
         await new DraftService().save(draft, user.bearerToken)
 
-        if (form.model.requestPermissionForExpert.option === YesNoOption.YES.option) {
+        if (draft.document.permissionForExpert.option.option === YesNoOption.YES.option) {
```
Comment (andrewwa-kainos) [inline src/main/features/directions-questionnaire/routes/permission-for-expert.ts]: `.option.option` looks like something's going wrong. Please double-check

File: src/main/features/directions-questionnaire/views/hearing-location.njk [hunk 5]
```diff
@@ -52,20 +52,20 @@
...
+                  checked: form.valueFor('courtAccepted').option === YesNoOption.NO.option
                 }
               ]
             ) }}
             <div class="js-hidden" id="courtAccepted-no">
               <h1 class="heading-medium">{{ t('Choose another court') }}</h1>
               {{ radioOption(form = form, label = 'Enter a court name', name = 'alternativeOption', value = 'name', checked = form.valueFor('alternativeOption') === 'name', bold = true) }}
               <div class="form-group panel panel-border-narrow js-hidden " id="alternativeOption-name" aria-hidden="false">
-                {{ textInput(label = 'Court name', form = form, name = 'alternativeCourtName', inputClass = 'full-width') }}
+                {{ textInput(label = 'Court name', form = form, name = 'alternativeCourtName', inputClass = 'full-width') }}Do you want to choose
```
Comment (andrewwa-kainos) [inline src/main/features/directions-questionnaire/views/hearing-location.njk]: Is that trailing "Do you want to choose" a copy/paste error?

File: src/test/features/directions-questionnaire/draft/directionsQuestionnaireDraft.ts [hunk 6]
```diff
@@ -1,9 +1,11 @@
 /* tslint:disable:no-unused-expression */
 import { expect } from 'chai'
 import { DirectionsQuestionnaireDraft } from 'directions-questionnaire/draft/directionsQuestionnaireDraft'
+import { sampleDirectionsQuestionnaireDraftObj } from '../../../http-mocks/draft-store'
```
Comment (andrewwa-kainos) [inline src/test/features/directions-questionnaire/draft/directionsQuestionnaireDraft.ts]: Refactor relative paths please

File: src/main/app/claims/models/directions-questionnaire/unavailableDate.ts [hunk 7]
```diff
@@ -0,0 +1,3 @@
+export interface UnavailableDate {
```
Comment (andrewwa-kainos) [inline src/main/app/claims/models/directions-questionnaire/unavailableDate.ts]: Why is this wrapper needed?

File: src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk (hunk 2 above)
Comment (robertparkinson) [reply src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk]: updated one of the others that was wrong.

**Comment by robertparkinson** [comment]
this PR is now redundant as the changes are done as part of this PR: https://github.com/hmcts/cmc-citizen-frontend/pull/1682

**Pull request closed without merging**